packages/validator/test/cli-validator/mock-files/oas3/err-and-warn.yaml

```diff
@@ -163,7 +163,7 @@ components:
       description: string
       required:
         - name
-        - photoUrls
+        - photo_urls
```

Comment:
This was causing a couple of errors to pop up due to Spectral reporting each path initially, so I just removed it. We didn't need it for the purpose of this test API definition